packages/in_app_purchase/lib/src/in_app_purchase/app_store_connection.dart

```diff
@@ -221,7 +221,7 @@ class _TransactionObserver implements SKTransactionObserverWrapper {
             ..error = transaction.error != null
```

Comment:
I wonder if we still need 
```
 ..status = SKTransactionStatusConverter()
                .toPurchaseStatus(transaction.transactionState)
            ..error = transaction.error != null
...
```
It seems everything has been moved to `fromSKTransaction`

Reply:
Removed.